src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -84,6 +87,12 @@ export class IeObjectsController {
 			maintainerId: user.getMaintainerId(),
 			accessibleObjectIdsThroughFolders: visitorSpaceAccessInfo.objectIds,
 			accessibleVisitorSpaceIds: visitorSpaceAccessInfo.visitorSpaceIds,
+			licensesByUserGroup:
+				IE_OBJECT_LICENSES_BY_USER_GROUP[
+					isNil(user.getId())
+						? IE_OBJECT_EXTRA_USER_GROUPS[IeObjectExtraUserGroupType.ANONYMOUS]
+						: user.getGroupId()
+				],
```

Comment:
Can't you just use this instead of IE_OBJECT_EXTRA_USER_GROUPS arrays?
```suggestion
				IE_OBJECT_LICENSES_BY_USER_GROUP[
					user.getGroupId() ?? IeObjectExtraUserGroupType.ANONYMOUS
				],
```